routes/api-routes.js

```diff
@@ -66,5 +66,19 @@ module.exports = function (app) {
     });
   });
 
-
-};
+  app.post("/api/create", function (req, res) {
```

Comment:
the newHero pull request will replace this app.post : it is safer and more robust